### PR TITLE
kms: updates website to document encryption config hot reload changes

### PR DESCRIPTION
**Pull request by nilekhc**
kms: updates website to document --encryption-provider-config-automatic-reload api server flag and healthcheck endpoint behavior

Signed-off-by: Nilekh Chaudhari <1626598+nilekhc@users.noreply.github.com>

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/kms-provider.md [hunk 1]
```diff
@@ -195,6 +197,8 @@ To encrypt the data:
              timeout: 3s
    ```
 
+Setting `--encryption-provider-config-automatic-reload` to `true` collapses all health checks to a single health check endpoint. Individual health checks are only available when KMS v1 providers are in used and encryption config is not auto-reloaded.
```
Comment (mengjiao-liu) [inline content/en/docs/tasks/administer-cluster/kms-provider.md:200]:
```suggestion
Setting `--encryption-provider-config-automatic-reload` to `true` collapses all health checks to a single health check endpoint. Individual health checks are only available when KMS v1 providers are in use and the encryption config is not auto-reloaded.
```

**Comment by divya-mohan0209** [comment]
Hey @nilekhc , from the issue description this looks like it is part of the 1.26 release. If it is, please could you rebase it to the dev-1.26 branch?

**Comment by leonardpahlke** [comment]
@nilekhc since we are targeting 1.26 could you take a look at the review? We need this merged in the next days, thx :)

File: content/en/docs/tasks/administer-cluster/kms-provider.md (hunk 1 above)
Comment (enj) [inline content/en/docs/tasks/administer-cluster/kms-provider.md:200]: Provide a table that enumerates all possible configs with what healthz checks are present.

i.e. with + without reloads vs. kms v1 only, kms v2 only, kms v1+v2, no kms

**Comment by jihoon-seo** [comment]
/remove-area blog release-eng
/remove-language de fr hi it pt ru zh
/remove-sig release

**Pull request closed without merging**